src/components/GNB/HomeLink.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect, useContext } from 'react';
+import Link from 'next/link';
+import * as Cookies from 'js-cookie';
+
+import { GNBContext } from 'src/components/GNB';
+import cookieKeys from 'src/constants/cookies';
+
+interface Props {
+  passHref?: boolean;
+  isPartials: boolean;
+}
+
+const legacyCookieMap = {
+  comic: 'comics',
+  romance_serail: 'romance-serial',
+  fantasy_serail: 'fantasy-serial',
+  bl_serail: 'bl-serial',
+};
+
+const HomeLink: React.FC<Props> = (props) => {
+  const {
+    passHref = false,
+    isPartials,
+    children,
+  } = props;
+
+  const { origin } = useContext(GNBContext);
+  const [genre, setGenre] = useState('');
+
+  useEffect(() => {
+    const cookie = Cookies.get(cookieKeys.main_genre);
+    setGenre(legacyCookieMap[cookie] ?? cookie);
+  });
```

Comment:
```suggestion
  }, []);
```
@tirr-c 이렇게 수정하면 될 것 같네요